datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -191,9 +206,46 @@ public void disconnect() {
         }
     }
 
+    private void activateDataSynchronisation() {
+        if (context.get() == null) {
+            throw new IllegalStateException("No valid context to enable data synchronization!");
+        }
+
+        Account account = getAccount();
+
+        boolean cyfaceAccountSyncIsEnabled = ContentResolver.getSyncAutomatically(account, AUTHORITY);
+        boolean masterAccountSyncIsEnabled = ContentResolver.getMasterSyncAutomatically();
+
+        if (cyfaceAccountSyncIsEnabled && masterAccountSyncIsEnabled) {
```

Comment:
If one of both accounts is not enabled, we do nothing, not even logging this event down?  
Can/should this happen anyway?

Reply:
These are not accounts but system settings, which the user uses to allow or disallow an App or all Apps to communicate with the internet. If it is not allowed we are out of luck an there really is nothing we can do.